cleverhans/attacks.py

```diff
@@ -210,6 +211,9 @@ def generate(self, x, **kwargs):
                   labels to avoid the "label leaking" effect (explained in this
                   paper: https://arxiv.org/abs/1611.01236). Default is None.
                   Labels should be one-hot-encoded.
+        :param y_target: (optional) A tensor with the labels to target. Do not
+                         set y_target if y is also set. Labels should be
```

Comment:
Maybe we can make the comment more useful by saying: If you want to run an untargeted (i.e., misclassification) attack while avoiding the label leaking effect, set y to the true labels and y_target to None, if you want to run a targeted (i.e., source-target misclassification attack), set y to None and y_target to the adversary's targets